projects/media-viewer/src/lib/annotations/annotation-set/annotation/rectangle/rectangle.component.html

```diff
@@ -4,7 +4,7 @@
        draggable
        resizable
        [tabindex]="rectangle.id"
-       [ngClass]="{ 'selected': selected }"
+       [ngClass]="{ 'selected': selected, 'dragScroll': enableDragScroll }"
```

Comment:
this is a general comment about the directive, why is it called dragScroll, it doesn't scroll, you grab and move around, so it's more of a grabDrag then dragScroll (I'm not saying call it grabDrag :smile:)

Reply:
grabDrag 🤔